actions/platform/chrome.py

```diff
@@ -0,0 +1,100 @@
+import json
+
+from actions.slot_match import FuzzySlotMatchOption
+from common.requests import send_console_request
+
+
+async def get_user(tracker):
+    # struggles to be a json response, manually doing it here
+    resp = await send_console_request(
+        "chrome-service",
+        "/api/chrome-service/v1/user",
+        tracker,
+        "get",
+    )
+
+    response, content = resp
+
+    return response, json.loads(content)
+
+
+async def create_service_options(tracker) -> list[FuzzySlotMatchOption]:
+    response, content = await get_generated_services(tracker)
+    if response.ok:
+        services = parse_generated_services(content)
+
+        options = []
+        for service in services:
+            if "href" not in service:
+                # path needs to be here to be favorited
+                continue
+            options.append(convert_service_to_option(service))
```

Comment:
Instead of an array of options, lets create a dict with the services - we will use them later to create the options we need

```suggestion
        options = {}
        for service in services:
            if "href" not in service:
                # path needs to be here to be favorited
                continue
            options[service["title"]] = convert_service_to_option(service)
```